hicommonwealth/commonwealth
# Remove Datadog Postgres config

## Link to Issue
Closes: #5872 

## Description of Changes
- Deletes `packages/commonwealth/datadog` -> redundant because of `DD_ENABLED_HEROKU_POSTGRES=true`

## Test Plan
- Deploy to Frick
- Ensure database metrics are still being reported to Datadog

## Deployment Plan

1. Remove the `DD_HEROKU_CONF_FOLDER` and `DATADOG_URL` env var -> @gdjohnson 

## Other Considerations

-
